bin/anthology/sigs.py

```diff
@@ -36,9 +37,7 @@
 
 def _sigevent_to_repr(event):
     if event.anthology_id is not None:
-        # For some reason, SIG files point to the front matter, not to the
-        # containing proceedings volume -- change that here
-        return to_volume_id(event.anthology_id)
+        return event.anthology_id
     return {"name": event.name, "url": event.url}
```

Comment:
SIG events used to list the frontmatter paper (e.g., `S19-1000`), but this was changed a while ago to list the volume directly.